package.json

```diff
@@ -15,7 +15,10 @@
     "build": "okta-sdk-generator -t templates/ -o . && ./utils/maintain-banners.js",
     "predocs": "rm -rf ./jsdocs && mkdir jsdocs/ && ./utils/make-jsdoc-readme.js > ./jsdocs/jsdoc-temp.md",
     "docs": "./node_modules/.bin/jsdoc src/ -c ./docs/config.json -d ./jsdocs/ -P ./package.json -R ./jsdocs/jsdoc-temp.md -r",
-    "test": "eslint -c src/.eslintrc src/ && eslint -c test/.eslintrc test/ && eslint -c src/.eslintrc utils/ && nyc --reporter=text --reporter=html mocha test/**/*.js --no-timeouts"
+    "lint": "eslint -c src/.eslintrc src/ && eslint -c test/.eslintrc test/ && eslint -c src/.eslintrc utils/",
```

Comment:
Doesn't need to be picked up with this PR, but we should remove the inclusive logic and instead omit directories we do not want `eslint` to run in.

Right now, it looks like we have multiple `eslint.rc` files with different rules.